docs/nuget-org/Publish-a-package.md

```diff
@@ -79,6 +79,19 @@ You can push packages to nuget.org with Azure Pipelines as part of your continuo
 
 To submit a signed package, you must first [register the certificate](../create-packages/Sign-a-Package.md#register-the-certificate-on-nugetorg) you used to sign the package. If you don't meet the [signed package requirements](../reference/Signed-Packages-Reference.md#signature-requirements-on-nugetorg), nuget.org rejects the package.
 
+### Package size limits
+
+Nuget.org has a package size limit of about 250 MB. When a package exceeding that limit is uploaded the following error is displayed:
```

Comment:
BTW, where do we get the package size limit? can we add size limit to the error message?